datafusion/sql/src/unparser/expr.rs

```diff
@@ -1595,46 +1706,6 @@ mod tests {
             ),
             (col("need-quoted").eq(lit(1)), r#"("need-quoted" = 1)"#),
             (col("need quoted").eq(lit(1)), r#"("need quoted" = 1)"#),
```

Comment:
It might be nice to leave a hint that date/time testing moved elsewhere
```suggestion
            (col("need quoted").eq(lit(1)), r#"("need quoted" = 1)"#),
            // See test_interval_scalar_to_expr for interval literals
```